bitnami/mongodb/templates/replicaset/statefulset.yaml

```diff
@@ -359,11 +359,15 @@ spec:
             - -ec
           args:
             - |
+              {{- if and .Values.tls.enabled .Values.auth.enabled }}
```

Comment:
Could we extract some repeated logic to a helper, something like:

```
{{- define "mongodb.mongodb_exporter.uri" -}}
    {{- $uriTlsArgs := ternary "tls=true&tlsCertificateKeyFile=/certs/mongodb.pem&tlsCAFile=/certs/mongodb-ca-cert" "" .Values.tls.enabled -}}
    {{- $uriAuth := ternary "root:$(echo $MONGODB_ROOT_PASSWORD | sed -r \"s/@/%40/g;s/:/%3A/g\")@" "" .Values.auth.enabled -}}

    {{- printf "mongodb://%slocalhost:27017/admin?%s" $uriAuth $uriTlsArgs -}}
{{- end -}}
```

Then use it in all the yamls like 

```
- |
   /bin/mongodb_exporter --mongodb.uri "{{ include "mongodb.mongodb_exporter.uri" . }}" {{ .Values.metrics.extraFlags }}
```

Reply:
Perfect I love this setup. I was trying to do it earlier but was unsure how to setup the helper. 
I just tested it and it works fine. 
And like I mentioned earlier, if I use the previous livenessProbe and readinessProbe command with this new helper with tls enabled it fails. However once I give it the string **"/bin/mongodb_exporter --mongodb.uri "{{ include "mongodb.mongodb_exporter.uri" . }}" {{ .Values.metrics.extraFlags }}
                  --test"** for the probes it works perfect.